src/declarations.d.tsx

```diff
@@ -1,4 +1,3 @@
 declare module '@carbon/react';
 declare module '*.css';
 declare module '*.scss';
-declare type SideNavProps = {};
```

Comment:
This was complaining citing duplication

Reply:
Hhhhmm.. I'm not sure what the problem was but we need this type declaration for clean builds